scripts/release.sh

```diff
@@ -74,6 +69,13 @@ if [ ! -z "$RELEASE_DRY_RUN" ]; then
 fi
 
 if [ "${npm_previous_package_version%.*}" != "${npm_current_package_version%.*}" ]; then
+    echo "New minor version - Updating the OldVersion.md file"
+    npm_previous_package_minor_version=${npm_previous_package_version%.*}
+    npm_current_package_minor_version=${npm_current_package_version%.*}
+    echo $npm_previous_package_minor_version
+    echo $npm_current_package_minor_version
+    sed -i "s/^- \[v$npm_previous_package_minor_version\].*/- [v$npm_current_package_minor_version](https:\/\/github.com\/marmelab\/react-admin\/blob\/master\/docs\/Admin.md)\n- [v$npm_previous_package_minor_version](https:\/\/github\.com\/marmelab\/react\-admin\/blob\/$npm_previous_package_version\/docs\/Admin.md\)/" docs/OldVersions.md
```

Comment:
Also a nitpick: this will be committed with the next `git commit` instruction under the message `"Update create-react-admin templates dependencies for version..."`.

IMO we should either
- Create a separate commit for updating the `OldVersions.md` file
- Update the commit message to be more general, and make it clearer (in the code indentation) this commit includes both actions